### ISSUE TITLE
Script error

### ISSUE THREAD

**Issue by devynosborne** [alexkuc/omnifocus-projects-script#1]
I get the following error when the script runs. 

<img width="429" alt="2020-02-23 - PM-05-20-56 - screen shot" src="https://user-images.githubusercontent.com/5923527/75124043-684e4d80-5661-11ea-8820-0e559b51fbab.png">

The script previously ran successfully, but now I get this error at the same point each time. The script gets the last successful project's name, but not any of the other info (eg, status); that's where it seems to stop. Since the last successful run, I have changed the name of that project, but I don't think I've changed anything else.

UPDATE: I realized I did change one thing: I updated OmniFocus. Since last run of this script, I believe I updated from 3.4.5 (perhaps earlier?) to 3.5.1 (which I'm currently running). Perhaps something in the most recent few releases changed something?

**Comment by alexkuc** [comment]
Hello, @devynosborne !

Have you tried running the latest version of the script? I have added some changes to the script on 12th of February in `OmniFocus3` branch. They contain some improvements as well as bug fixes.

~~Edit: After reading more carefully the screenshot with the error message, I have realised which part of the script fails so I have contacted OmniGroup support for their assistance. Once I have an update and the solution, I will update this issue as well as commit the fix.~~
